docs/conf.py

```diff
@@ -18,9 +18,12 @@
 
 import os
 import sys
-from smrf.utils.utils import get_config_doc_section_hdr
+
 from inicheck.tools import config_documentation
+# -- Have to do a mock install of some modules that RTD doesn't have --------
+from mock import Mock as MagicMock
```

Comment:
Should the 'mock' lib go into the dev requirements as well?

Reply:
I didn't realize that this library is part of `unittest.mock` now. I just changed that to reflect and removed from the requirements.